README.md

```diff
@@ -25,21 +25,21 @@ see the [Installing section in the user guide](https://www.montepy.org/starting.
 ## User Documentation
 
 MontePy has a [sphinx website](https://www.montepy.org/). 
-This has a getting started guide for users,
-as well as API documentation. 
-There is also a developer's guide covering the design and approach of MontePy, and how to contribute.
+The website contains a user's guide for getting started, 
+a developer's guide covering the design and approach of MontePy,
+instructions for contributing, 
+and the Python API documentation.
 
 ## Features
 	
-* Handles almost all MCNP input syntax including: message blocks, & continue, comments, etc.
-* Parses Cells, surfaces, materials, and transforms very well.	
-* Can parse the following surfaces exactly P(X|Y|Z), C(X|Y|Z), C/(X|Y|Z) (I mean it can do PX, and PY, etc.)
-* Can read in all other inputs but not understand them	
-* Can write out full MCNP problem even if it doesn't fully understand an input.	
-* Can write out the MCNP problem verbatim, and try to match the original user formatting. 
+* Handles almost all MCNP input syntax.
+* Parses Cells, Surfaces, Materials, and Transforms very well.	
+* Can parse all surface types except macrobody facets ([Issue #354](https://github.com/idaholab/MontePy/issues/354)).
+* Can read and write out all other MCNP inputs, even if it doesn't not understand them	
+* Attempts to write out the MCNP problem verbatim, even matching the original user formatting. (See [Issues](https://github.com/idaholab/MontePy/issues).)
```

Comment:
Issues is to broad I think.

Reply:
I'm thinking leave the link, but change the text to say "See open issues"